imessage/mac-nosip/nosip.go

```diff
@@ -103,12 +105,12 @@ func (mac *MacNoSIPConnector) pingLoop(ipcProc *ipc.Processor) {
 			mac.log.Fatalln("Failed to send ping to Barcelona")
 			os.Exit(254)
 		}
-		timeout := time.After(15 * time.Second)
+		timeout := time.After(mac.pingInterval * time.Second)
 		select {
 		case <-mac.stopPinger:
 			return
 		case <-timeout:
-			mac.log.Fatalln("Didn't receive pong from Barcelona within 15 seconds")
+			mac.log.Fatalfln("Didn't receive pong from Barcelona within %s", mac.pingInterval)
```

Comment:
```suggestion
			mac.log.Fatalfln("Didn't receive pong from Barcelona within %s seconds", mac.pingInterval)
```

Reply:
Wouldn't the s be appended during formatting?

Reply:
Yeah it should add the time unit automatically

Reply:
Hmm... not actually sure how durations are formatted...